merlion/models/forecast/ets.py

```diff
@@ -197,11 +211,23 @@ def _forecast(
                 self.model = new_model.fit(start_params=self.model.params, disp=False)
             else:
                 self.model = new_model.smooth(params=self.model.params)
-            forecast_result = self.model.get_prediction(
-                start=val_prev.shape[0], end=val_prev.shape[0] + len(time_stamps) - 1, method="simulated"
-            )
-            forecast = np.asarray(forecast_result.predicted_mean)
-            err = np.asarray(forecast_result._results.simulation_results.std(axis=1))
+
+            # Some variants of ETS model does not support prediction interval when pred_interval_strategy="exact".
+            # In this case we use point forcasting and set prediction_interval as None
+            try:
+                forecast_result = self.model.get_prediction(
+                    start=val_prev.shape[0], end=val_prev.shape[0] + len(time_stamps) - 1,
+                    method=self.config.pred_interval_strategy
+                )
+                forecast = np.asarray(forecast_result.predicted_mean)
+                err = np.sqrt(np.asarray(forecast_result.var_pred_mean))
+            except NotImplementedError as v:
+                logger.warning(f"Caught exception {type(v).__name__}: {str(v)}")
+                forecast_result = self.model.predict(
+                    start=val_prev.shape[0], end=val_prev.shape[0] + len(time_stamps) - 1,
+                )
+                forecast = np.asarray(forecast_result)
+                err = None
```

Comment:
It's probably better to do `err = np.full(len(forecast), np.nan)`. This is less likely to cause errors in downstream code.